x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/CcrRepositoryIT.java

```diff
@@ -433,10 +424,6 @@ public void testFollowerMappingIsUpdated() throws IOException {
         clusterStateRequest.clear();
         clusterStateRequest.metaData(true);
         clusterStateRequest.indices(followerIndex);
-        ClusterStateResponse clusterState = followerClient().admin().cluster().state(clusterStateRequest).actionGet();
-        IndexMetaData followerIndexMetadata = clusterState.getState().metaData().index(followerIndex);
-        assertEquals(2, followerIndexMetadata.getMappingVersion());
```

Comment:
why remove this?

Reply:
Previously we changed mapping twice in the leader: one before opening session and one after. These two events are propagated separately; then the mapping version on the follower is 2. Now, these two events happen before we open session, then the mapping version on the follower is 1 instead of 2. I think verifying the content of the mapping is good enough.

Reply:
ok